app/controllers/CollectionController.scala

```diff
@@ -8,6 +8,7 @@ import services.Press
 import updates._
 import util.Acl
 import util.Requests._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
This will work fine but I think we should go a step further and add the execution context as an implicit parameter on the constructor of the controllers. It will then get filled in with the implicit `executionContext` Play constructs in `BuiltInComponents`.

Same goes for all the global context imports below.